pom.xml

```diff
@@ -1,163 +1,162 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>org.dbpedia.spotlight</groupId>
-  <artifactId>wikipedia-stats-extractor</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <licenses>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.dbpedia.spotlight</groupId>
+    <artifactId>wikipedia-stats-extractor</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <licenses>
         <license>
             <name>The Apache Software License, Version 2.0</name>
             <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
         </license>
-  </licenses>
-  <properties>
-    <scala.version>2.10.4</scala.version>
-      <project.source.charset>UTF-8</project.source.charset>
-      <project.source.encoding>UTF-8</project.source.encoding>
-      <java.version.source>1.7</java.version.source>
-      <java.version.target>1.7</java.version.target>
-  </properties>
+    </licenses>
+    <properties>
+        <scala.version>2.10.4</scala.version>
+        <project.source.charset>UTF-8</project.source.charset>
+        <project.source.encoding>UTF-8</project.source.encoding>
+        <java.version.source>1.7</java.version.source>
+        <java.version.target>1.7</java.version.target>
+    </properties>
 
-  <repositories>
-    <repository>
-      <id>scala-tools.org</id>
-      <name>Scala-Tools Maven2 Repository</name>
-      <url>http://scala-tools.org/repo-releases</url>
-    </repository>
-      <repository>
-          <id>maven.aksw.internal</id>
-          <name>University Leipzig, AKSW Maven2 Repository</name>
-          <url>http://maven.aksw.org/archiva/repository/internal</url>
-      </repository>
-  </repositories>
+    <repositories>
+        <repository>
+            <id>scala-tools.org</id>
+            <name>Scala-Tools Maven2 Repository</name>
+            <url>http://scala-tools.org/repo-releases</url>
+        </repository>
+        <repository>
```

Comment:
Yes, i am using few classes from extraction repo for creating DBpedia uri. So would the repo be removed in future? What would be the long term solution in your opinion.. Since i am using spotlight and few classes from extraction.